<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>текстовый флаг</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="tabletTracking">
      <bool>false</bool>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="verticalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>200</y>
        <width>121</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QRadioButton" name="Button_radio_1_1">
         <property name="text">
          <string>синий</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="Button_radio_1_2">
         <property name="text">
          <string>красный</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="Button_radio_1_3">
         <property name="text">
          <string>зелёный</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>200</y>
        <width>131</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QRadioButton" name="Button_radio_2_1">
         <property name="text">
          <string>синий</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="Button_radio_2_2">
         <property name="text">
          <string>красный</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="Button_radio_2_3">
         <property name="text">
          <string>зелёный</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>200</y>
        <width>141</width>
        <height>80</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QRadioButton" name="Button_radio_3_1">
         <property name="text">
          <string>синий</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="Button_radio_3_3">
         <property name="text">
          <string>красный</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="Button_radio_3_4">
         <property name="text">
          <string>зелёный</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="Button_main">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>320</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Сделать флаг</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_answer">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>420</y>
        <width>311</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="mouseTracking">
       <bool>false</bool>
      </property>
      <property name="text">
       <string></string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>141</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Цвет №1</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>120</y>
        <width>141</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Цвет №2</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>120</y>
        <width>141</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>25</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Цвет №3</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
